pom.xml

```
@@ -38,7 +38,7 @@
     <packaging>hpi</packaging>
 
     <name>CloudBees Docker Hub/Registry Notification</name>
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Hub+Notification</url>
+    <url>https://plugins.jenkins.io/dockerhub-notification/</url>
```

```suggestion
    <url>https://github.com/jenkinsci/dockerhub-notification-plugin/</url>
```
Otherwise the move won't be correct, since this is the magic that the plugin site reads to see where to get the docs from.